state-sync/src/logging.rs

```diff
@@ -17,7 +17,7 @@ pub struct LogSchema<'a> {
     name: LogEntry,
     event: Option<LogEvent>,
     #[schema(debug)]
-    error: Option<&'a anyhow::Error>,
+    error: Option<&'a anyhow::Error>, // TODO(joshlind): remove anyhow once we migrate!
```

Comment:
Just personally, I think it's probably better to track this kind of stuff in github issues as high level problems.  E.g. remove `anyhow` from state-sync.  You'll have to come back through all of these anyways

Reply:
Yeah, I already have this as an item in the issue: https://github.com/diem/diem/issues/6795